src/ex1/script.js

```diff
@@ -1,50 +1,111 @@
 const input = document.querySelector(".todo-input");
-const addButton = document.querySelector(".todo-button");
+const addButton = document.querySelector(".add-todo-button");
+const deleteAll = document.querySelector(".todo-button-delete-all");
 const todoList = document.querySelector(".todo-list");
+let COUNTER = 0;
```

Comment:
we use upper case letters when its a const variable (variable that his valus never changed)
```suggestion
let itemsCount = 0;
```